data/advertisements.csv

```diff
@@ -1,2 +1,2 @@
 "id","type","format","advertiser","price","price_type","url","title","details"
-"1","image","medium","Example company","0.5 USD","PER_VIEW","https://example.ads.com","Some title","Some details"
+"1","IMAGE","MEDIUM","Example company","0.5 USD","PER_VIEW","https://example.ads.com","Some title","Some details"
```

Comment:
to nie jest konieczne jbc bo jest zrobione equalsIgnoreCase(), ale to just sayin

Reply:
ok